hack/manifest-gen/assets/charts/eck/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: eck
+description: A Helm chart for deploying Elastic Cloud on Kubernetes
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.0.1
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application.
+appVersion: 1.2.0
```

Comment:
IIUC this field should be kept in sync with https://github.com/elastic/cloud-on-k8s/blob/master/VERSION ?

Reply:
It is: https://github.com/charith-elastic/cloud-on-k8s/blob/6f3f2bee22fff6742d29b9d5e5b0685cad3a6186/hack/manifest-gen/manifest-gen.sh#L14